src/consumer/component/sources/files.js

```diff
@@ -54,12 +52,12 @@ export default class Files extends Source {
         return Promise.resolve();
```

Comment:
This still using map and promise all on no longer an array.
Please remove the unnecessary code (you won't see it here in the PR because you didn't change it)